src/include/private/switch_core_pvt.h

```diff
@@ -170,6 +170,8 @@ struct switch_core_session {
 	switch_buffer_t *text_line_buffer;
 	switch_mutex_t *text_mutex;
 	const char *external_id;
+
+	packet_stats_t stats;
```

Comment:
This should not be here. We have  RTP stats on the session through media_handle -> RTP engines.
See switch_rtp_stats_t stats in the rtp_session.

Reply:
thanks, will look at it